assets/kill-debian.sh

```diff
@@ -0,0 +1,74 @@
+#!/data/data/info.guardianproject.lildebi/app_bin/sh
+#
+# see lildebi-common for arguments, the args are converted to vars there. The
+# first arg $1 is the "app payload" directory, where the included scripts are
+# kept.
+
+# many phones don't even include 'test', so set the path to our
+# busybox tools first, where we provide all the UNIX tools needed by
+# this script
+export PATH=$1:$PATH
+
+test -e $1/lildebi-common || exit 1
+. $1/lildebi-common
+
+echo -n "Asking all processes to terminate..."
+# first send TERM
+for root in /proc/*/root; do
+  if [ ! -r "$root" ] || [ ! "`readlink "$root"`" = "$mnt" ]; then
+    continue
+  fi
+  pid="${root#/proc/}"
+  pid="${pid%/root}"
+  kill -TERM "$pid" 2>/dev/null || true
+done
+# /debian/shell starts outside of the chroot, so it must be killed separately
+for pid in `ps | grep '/[d]ebian/shell' | cut -b1-5`; do
+    kill -TERM $pid
+done
+sleep 1
+echo "done"
+# then send KILL to force the rest to die
+echo "Killing remaining processes:"
+for root in /proc/*/root; do
+  if [ ! -r "$root" ] || [ ! "`readlink "$root"`" = "$mnt" ]; then
+    continue
+  fi
+  pid="${root#/proc/}"
+  pid="${pid%/root}"
+  echo "$pid `readlink $(dirname $root)/exe`"
+  kill -KILL "$pid" 2>/dev/null || true
+done
+for pid in `ps | grep '/[d]ebian/shell' | cut -b1-5`; do
```

Comment:
Small suggestion: use `grep '/debian/shell' /proc/*/comm` instead of parsing ps output.